salt/templates/virt/libvirt_domain.jinja

```diff
@@ -51,7 +54,11 @@
                             {% endif %}/>
                 </graphics>
                 {% endif %}
-                {% if serial_type == 'pty' %}
+		{% if serial_type == 'nmdm' %}
+		<serial type='nmdm'>
+			<source master='/dev/nmdm{{ nmdm_id }}A' slave='/dev/nmdm{{ nmdm_id }}B'/>
+		</serial>
+                {% elif serial_type == 'pty' %}
```

Comment:
The indentation is off here. Please adjust to match the surrounding code.

Reply:
I'ld love to see some unit tests for the new serial type and the loader element to avoid breaking the features you need with future potential refactoring.